android/java/org/chromium/chrome/browser/settings/BraveMainPreferencesBase.java

```diff
@@ -73,7 +73,6 @@
     private static final String PREF_PRIVACY = "privacy";
     private static final String PREF_SHIELDS_AND_PRIVACY = "brave_shields_and_privacy";
     private static final String PREF_BRAVE_SEARCH_ENGINES = "brave_search_engines";
-    private static final String PREF_BRAVE_NEWS = "brave_news";
     private static final String PREF_BRAVE_NEWS_V2 = "brave_news_v2";
```

Comment:
whoops - is this the wrong string removed^^? Removing `brave_news` instead of `brave_news_v2`?

Reply:
Nah, this is right because we've removed the V2 flag (v2 is always on), so none of the V1 stuff is used anymore